cython/bvh.pyx

```diff
@@ -0,0 +1,64 @@
+import random
+include "sphere.pyx"
+
+cdef class BoundingVolumeHierarchyNode(Hittable):
```

Comment:
For this class, we have an equivalent Python implementation already. It works well and is not performance critical,
so well keep using that one, as the Cython class is much harder to maintain